src/fs/normalizer/header.ts

```diff
@@ -76,10 +73,15 @@ export const getChildKey = async (cid: CID, key: string): Promise<string> => {
  * ```
  */
 export const put = async (index: CID, header: Header, key: Maybe<string>): Promise<CID> => {
-  const noUndefined = _.pickBy(header, isDefined)
+  const noUndefined = Array.from(
```

Comment:
I.... thought I tried this and typescript didn't like whatever I did :thinking: 

Ah well, if it works then I'm happy to eliminate the dependency!

Reply:
I had to add `val as any` a few lines further down 🤷

Reply:
ah yup didn't see that. hmmm I really like to avoid `any`s, altho this one is fairly contained. I wonder why TS won't cooperate :thinking:

Reply:
Managed to replace `as any` with `as FileContent`, which fixes it 👍

Reply:
awww yisss